src/frontend/screens/Settings/ProjectConfig.tsx

```diff
@@ -114,26 +117,85 @@ const ProjectConfig = () => {
         <Text style={[styles.centerText, styles.projectKey]}>
           {projectKeySlice ? projectKeySlice + "**********" : "MAPEO"}
         </Text>
+
+        {!!configName && (
+          <React.Fragment>
+            <Text style={[styles.centerText, { marginTop: 10 }]}>
+              <FormattedMessage {...m.name} />
+            </Text>
+            <Text style={[styles.centerText, styles.projectKey]}>
+              {configName || ""}
+            </Text>
+          </React.Fragment>
+        )}
         {(status === "loading" || config.status === "loading") && (
           <View style={styles.loadingContainer}>
             <ActivityIndicator size="large" />
           </View>
         )}
       </View>
+
       <Button
         disabled={status === "loading" || config.status === "loading"}
-        variant="outlined"
+        variant="contained"
         onPress={handleImportPress}
+        style={[{ marginTop: 40 }, styles.button]}
       >
         {t(m.importConfig) /* Button component expects string children */}
       </Button>
+      {process.env.FEATURE_ONBOARDING === "true" && (
+        <React.Fragment>
+          {/* We need to work out permission before we figure out what this button does */}
+          <Button
+            style={styles.button}
+            disabled={status === "loading" || config.status === "loading"}
+            variant="outlined"
+            onPress={() => {}}
+          >
+            <View style={styles.LeaveBttn}>
+              <MaterialIcons name="person-add" color="#0066ff" size={18} />
+              <Text
+                style={{
+                  color: "#0066ff",
+                  fontWeight: "bold",
+                  marginLeft: 5,
+                  fontSize: 16,
+                }}
+              >
+                <FormattedMessage {...m.addPerson} />
+              </Text>
+            </View>
+          </Button>
+
+          <Button
+            style={styles.button}
+            disabled={status === "loading" || config.status === "loading"}
+            variant="outlined"
+            onPress={() => navigate("LeaveProjectScreen")}
+          >
+            <View style={styles.LeaveBttn}>
+              <MaterialIcons name="exit-to-app" color="#0066ff" size={18} />
+              <Text
+                style={{
+                  color: "#0066ff",
+                  fontWeight: "bold",
+                  marginLeft: 5,
+                  fontSize: 16,
+                }}
+              >
+                <FormattedMessage {...m.leaveProject} />
+              </Text>
+            </View>
+          </Button>
+        </React.Fragment>
+      )}
     </View>
   );
 };
 
 ProjectConfig.navigationOptions = {
   headerTitle: () => (
-    <HeaderTitle>
+    <HeaderTitle style={{}}>
```

Comment:
is this necessary?

Reply:
Its throwing an error if I dont. And I didnt want to spend time converting the component to typescript (its not a simple conversion) I could also ts-ignore it, but I thought this was nicer